spec/ttml3.xml

```diff
@@ -16633,7 +16693,7 @@ element type of <emph>E</emph> is not animation element type <el>animate</el> or
 and is not styling element type <el>style</el>,
 then for each style property
 <emph>P</emph> in the set of style properties defined
-above in <specref ref="styling-attribute-vocabulary"/>, perform the
+in <specref ref="styling-attribute-vocabulary"/> or in a supported <loc href="#terms-external-module">external module</loc>, perform the
```

Comment:
I think we need something a bit stronger here, in addition to this. Specifically, we need to define a class of attributes considered to be "style" attributes, which a module may declare, such that the set of "style" attributes defined by supported modules is the set which are included in the style resolution process, with inheritance, computation, style set merging etc.

Reply:
At present, we define [style property] in 10.4.4.1, but we do not constrain it (this definition) regarding which namespace is used. On the other hand, on various occasions, we do constrain a style property to one defined in one of the TT Style Namespaces.

I would suggest we take up this comment in a separate issue, and go ahead and accept the current language here, since it is consistent with many other uses in the spec.

Reply:
That works for me.

Reply:
I raised #32.